docs/spec/options/hurl/max_filesize.option

```diff
@@ -0,0 +1,9 @@
+name: max_filesize
+long: max_filesize
+value: NUM
+value_default: 0
+value_parser: clap::value_parser!(u64)
+help: File size limits
+cli_only: true
+---
+Limit filesize in kb that you allow a response to take.
```

Comment:
Unit should be `bytes`. Maybe we can take the wording from curl: `Specify the maximum size (in bytes) of a file to download`

Reply:
`cli_only` is here to specify if we want this option to appear in `[Options]` section. As we want to write this kind of file:

```
GET https://foo.com
[Options]
max-filesize: 32000
HTTP 200
```

Reply:
Also `long: max_filesize` should `long: max-filesize`